ext/mbstring/tests/bug43994.phpt

```diff
@@ -49,7 +49,8 @@ With $regs arg:
 
 Warning: mb_ereg(): empty pattern in %s on line %d
 bool(false)
-NULL
+array(0) {
+}
```

Comment:
> I don't think changing ereg() makes sense, as it has been deprecated as of PHP 7.0.0.

Actually, it had already been deprecated as of PHP 5.3.0, and *removed* as of PHP 7.0.0.